src/content/docs/en/reference/api-reference.mdx

```diff
@@ -778,10 +826,79 @@ const draftBlogPosts = await getCollection('blog', ({ data }) => {
 
 [See the `Content Collection` guide](/en/guides/content-collections/#querying-collections) for example usage.
 
+### `getEntry()`
+
+<p>
+  <Since v="2.5.0" />
+</p>
+
+**Types:**
+
+- `(collection: string, contentSlugOrDataId: string) => CollectionEntry<collection>`
+- `({ collection: string, id: string }) => CollectionEntry<collection>`
+- `({ collection: string, slug: string }) => CollectionEntry<collection>`
+
+`getEntry()` is a function that retrieves a single collection entry by collection name and either the entry `id` (for JSON or YAML collections) or entry `slug` (for Markdown or MDX collections).
+
+```astro
+---
+import { getEntry } from 'astro:content';
+
+// Get `src/content/blog/enterprise.md`
+const enterprisePost = await getEntry('blog', 'enterprise');
+
+// Get `src/content/captains/captain-kirk.yaml`
+const captainProfile = await getEntry('captains', 'captain-kirk');
+---
+```
+
+`getEntry()` can also be used to get referenced entries to access the `data`, `body`, or `render()` properties:
+
+```astro
+---
+import { getEntry } from 'astro:content';
+
+const enterprisePost = await getEntry('blog', 'enterprise');
+
+// Get the profile referenced by `data.captain`
+const captainProfile = await getEntry(enterprise.data.captain);
+---
+```
+
+See the `Content Collection` guide for examples of [querying](/en/guides/content-collections/#querying-collections) and [references](/en/guides/content-collections/#resolving-references).
```

Comment:
```suggestion
See the `Content Collections` guide for examples of [querying collection entries](/en/guides/content-collections/#querying-collections).
```
References is included in the querying section, and in fact, I have another note coming that I'm not entirely sure `resolving-references` needs to be its own section in the querying section anyway.  :smile: